test/define-test.js

```diff
@@ -17,3 +17,85 @@ QUnit.test("Can define stuff", function(assert) {
   let faves = new Faves();
   assert.equal(faves.color, "blue", "Got the value");
 });
+
+QUnit.test("Can not define stuff", function(assert) {
+	class Faves extends dFine() {}
+
+	let faves = new Faves();
+	assert.ok(true, "created the instance");
+});
+
+QUnit.test("Stuff is defined in constructor for non-element classes", function(assert) {
+  class Faves extends dFine(Object) {
+    static get define() {
+      return {
+		  color: {
+			  default: "blue"
+		  }
+      };
+    }
+
+	constructor() {
+		super();
+		assert.equal(this.color, "blue", "color exists after constructor");
+	}
+  }
+
+  let faves = new Faves();
+});
+
+QUnit.test("Stuff is defined in connectedCallback for custom elements", function(assert) {
```

Comment:
Saucelabs is failing in Edge, I'm guessing for a lack of custom element support.  I would detect CE support and only run these test in supported browsers.

Reply:
`'customElements' in window` should be enough.

Reply:
Should we just add the polyfill? That's what I'm doing in `can-stache-define-element`.

Reply:
I don't think its appropriate in this case, can-define-class is not about creating custom elements. This is just testing that it works correctly when used with custom elements.